### PR TITLE
Fix docs links

### PR DESCRIPTION
**Pull request by vvbandeira**

### CODE REVIEW

**Comment by vvbandeira** [comment]
@luarss FYI

File: docs/index.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ linked [here](https://theopenroadproject.org/publications/).
 
 ## Code of conduct
 
-Please read our code of conduct [here](../CODE_OF_CONDUCT.md).
+Please read our code of conduct [here](main/CODE_OF_CONDUCT.md).
```
Comment (vijayank88) [inline docs/index.md:42]: Page not found

**Review by vijayank88 — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The application is a standalone binary capable of performing RTL-to-GDSII SoC de
 from logic synthesis and floorplanning through detailed routing with metal fill insertion,
 signoff parasitic extraction and timing analysis.
 
-See documentation for the application [here](../README.md).
+See documentation for the application [here](main/README.md).
```
Comment (vijayank88) [inline docs/index.md:54]: Page not found

**Review by vijayank88 — CHANGES_REQUESTED** [review]

**Comment by vijayank88** [comment]
Other document changes LGTM...

File: docs/index.md (hunk 2 above)
Comment (vvbandeira) [reply docs/index.md:54]: this will be resolved correctly in readthedocs.

**Pull request merged**